src/containers/shared/components/TokenTableRow.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export interface TokenTableRowProps {
+  shouldDisplay?: boolean
+  label: string
+  value: any
+}
+
+export const TokenTableRow = (props: TokenTableRowProps) => {
+  const { shouldDisplay = true, label, value } = props
+  return (
+    <>
+      {shouldDisplay && (
```

Comment:
Why not just make this
```suggestion
      {value && (
```

Reply:
Yeah good point, I thought some of these conditions were independent of the value but there wasnt. Will change it

Reply:
oh the `is_burned` field is a boolean, but I cant print that in, so I had to pass in a  string